doc/contributing/docs/style.rst

```diff
@@ -174,13 +174,38 @@ Spelling and punctuation
 US vs British spelling
 ~~~~~~~~~~~~~~~~~~~~~~
 
-We use the US English spelling.
+Use the US English spelling.
 
 Check your spelling and punctuation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Consider checking spelling, grammar, and punctuation with special tools.
 
+Dashes
+~~~~~~
+
+Special symbols like dashes, quotation marks, and apostrophes look the same
+across all Tarantool documentation in a single language.
+This is because the documentation :doc:`builder </contributing/docs/build/>`
+renders specific character sequences in the source into correct typographic characters.
+
+Type three hyphens (``---``) to insert an em dash (---)
+and two hyphens (``--``) for an en dash (--).
+The following rules apply:
+
+*   The shorter **en dash** ``--`` is used to mark ranges (for example, 4--16 GB).
+*   The longer **em dash** ``---`` is used to separate extra information or mark a break in a sentence.
+
+Don't use a single hyphen as a dash. Don't add spaces on either side of a dash. 
```

Comment:
As I see, both styles are in use in English (with and without surrounding whitespaces). In my opinion, em dash without surrounding spaces looks odd. So I'm curious, what is the reason behind choosing the style without spaces.

Reply:
Originally, the em dash was surrounded by typographic marks known as "thin spaces". But teaching our users more about types of spaces (so many of them) is just cruel IMO. We can either drop the spaces or make them thicker. I chose the former option because this way the usage is similar to the en dash, so the user has less to think about.

Reply:
Couldn't it be solved automatically?

@NickVolynkin ?